clients/unshaded/src/main/java/tachyon/client/TachyonFile.java

```diff
@@ -495,7 +495,7 @@ boolean recache(int blockIndex) throws IOException {
       inputStream.skip(offset);
 
       int bufferBytes =
-          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB) * 4;
+          (int) mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB);
```

Comment:
I think it should be 1 MB given the default property value file. You can open another JIRA ticket to investigate what size is ideal in most situations, but I think it should be part of a separate PR.